hooks/helmlint.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# We don't set `set -e` in this script because some of the functions will return a non-zero value (e.g contains_element)
```

Comment:
See the suggested if-statement below, which, even with `set -e`, does not cause the whole script to exit.

Reply:
To put it another way, it would be better to explicitly handle the cases where non-zero codes are expected than to implicitly allow them everywhere, including places you don't expect them.